package.json

```diff
@@ -5,15 +5,11 @@
     "@babel/cli": "^7.1.5",
     "@babel/core": "^7.2.0",
     "@babel/plugin-proposal-class-properties": "^7.2.1",
-    "@babel/plugin-proposal-export-default-from": "^7.0.0",
     "@babel/plugin-syntax-dynamic-import": "^7.2.0",
     "@babel/plugin-transform-modules-commonjs": "^7.2.0",
-    "@babel/plugin-transform-runtime": "^7.2.0",
-    "@babel/preset-env": "^7.1.6",
     "@babel/preset-flow": "^7.0.0",
     "@babel/preset-react": "^7.0.0",
     "@mdx-js/loader": "^0.16.6",
-    "@mdx-js/mdx": "^0.16.6",
```

Comment:
we are still using `mdx-js` loader in webpack.config.js right?

Reply:
Yep!